requirements.txt

```diff
@@ -10,3 +10,4 @@ tox==3.24.5
 pandas==1.5.1
 plotly==5.11.0
 kaleido==0.2.1
+freesasa==2.2.0.post3
```

Comment:
I don't see any information in their documentations about this `post` version, can you use `2.2.0` instead?

Reply:
probably yes!

Reply:
tried with 2.2.0 and 2.1.0, both failed for some hyper-weird reason..I would stick to this version for the time being